### PR TITLE
OH2-188 Add parameter status on api getLaboratoryForPrint

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OH2-188.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+OH_MARIADB_PORT=3306
+OH_MARIADB_DATABASE=oh
+OH_MARIADB_ROOT_PASSWORD=root
+OH_MARIADB_USER=isf
+OH_MARIADB_PASSWORD=isf123
+OH_DEMO_LANG=en
+
+OH_CORE_BRANCH=develop
+OH_API_BRANCH=develop
+GITHUB_ORG=intesys
```
Comment (dbmalkovsky) [inline .env:11]: Is this the correct repo?   And shouldn't there be some Java code here too?

Comment (mwithi) [reply .env:11]: I don't think this PR has any meaning as for now

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 2]
```diff
@@ -521,13 +520,15 @@ public ResponseEntity<List<String>> getMaterials() throws OHServiceException {
 	public ResponseEntity<List<LabWithRowsDTO>> getLaboratoryForPrint(
 			@RequestParam(required = false, defaultValue = "") String examName,
 			@RequestParam(value = "dateFrom") String dateFrom, @RequestParam(value = "dateTo") String dateTo,
-			@RequestParam(value = "patientCode", required = false, defaultValue = "0") int patientCode)
+			@RequestParam(value = "patientCode", required = false, defaultValue = "0") int patientCode,
+			@RequestParam(value = "status", required = false, defaultValue = "") String status)
 			throws OHServiceException {
 		LOGGER.info("Get labWithRow within specified date");
 		LOGGER.info("examName: {}", examName);
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:527]: I would put only first logging as INFO, all others as DEBUG

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**